core-bundle/contao/dca/tl_content.php

```diff
@@ -965,7 +965,15 @@ public function addCteType($arrRow)
 		$objModel->setRow($arrRow);
 
 		$class = 'cte_preview';
-		$preview = StringUtil::insertTagToSrc($this->getContentElement($objModel));
+
+		try
+		{
+			$preview = StringUtil::insertTagToSrc($this->getContentElement($objModel));
+		}
+		catch (Throwable $exception)
```

Comment:
```suggestion
		catch (\Throwable $exception)
```
Just a nit :)

Reply:
This file has no namespace and our coding style is to not prefix the root-namespace classes in that case. (at least that is how the other root-namespace classes are referenced in this file already).